src/reference/sectionA_definitions.inc

```diff
@@ -105,7 +105,7 @@ CellML and XML
 Specific information items
 ~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-#. For the purposes of this specification, a specific information item is one of the following (see https://www.w3.org/TR/xml-infoset/#infoitem for definitions):
+#. For the purposes of this specification, a specific information item is one of the following (see https://www.w3.org/TR/xml-infoset/ for definitions):
 
```

Comment:
I should have done this already ... but it would be good to have this included in the external links section in the config file?  Makes it easier to see what we're using if they're all listed there?  We can still write out the address here though.

Reply:
We have the :xml_infoset: tag already (see below).  Should this be something less specific??  It's currently only used at the beginning of the Terminology section (second sentence of entire doc):
'xml_infoset': ('http://www.w3.org/TR/2004/REC-xml-infoset-20040204/%s', ''),

Reply:
pretty sure this change was just to link to the top of the page as the table of contents gives a better "overview" of the different types of info items. The other links are to specific items, so the more specific link is correct.

Reply:
Making separate issue for this!